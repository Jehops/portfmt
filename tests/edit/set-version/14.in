PORTVERSION=	1.0
PORTREVISION?=	10
