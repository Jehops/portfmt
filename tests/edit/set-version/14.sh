${PORTEDIT} set-version 1.1 14.in | diff -L 14.expected -L 14.actual -u 14.expected -
