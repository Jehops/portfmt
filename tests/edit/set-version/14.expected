PORTVERSION=	1.1
PORTREVISION?=	0
